src/ai/backend/common/service_discovery/service_discovery.py

```diff
@@ -60,7 +60,7 @@ class ServiceMetadata(BaseModel):
     Metadata for a service.
     """
 
-    id: uuid.UUID
+    id: uuid.UUID = Field(default_factory=uuid.uuid4)
```

Comment:
Seems service metadata can be added when the service restarts. How can we clean up the outdated IDs?

Reply:
By default, services will deregister on exit, but even if they don't, services that haven't had a heartbeat for a long time will be cleaned up automatically.